src/buildList.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+const arrayOfBooks = 
+[
+	{id: 10001,
+	name: "Mr. Penumbras 24-Hour Bookstore",
```

Comment:
Use single quotes when you use simple string, it's not a rule but it is common usage on the most of projects https://stackoverflow.com/questions/242813/when-to-use-double-or-single-quotes-in-javascript